root/etc/e-smith/events/actions/nethserver-nextcloud-conf

```diff
@@ -6,13 +6,11 @@ function OCC
     params=$@;
     source /opt/rh/rh-php73/enable
     cd /usr/share/nextcloud/
-    TERM=dumb runuser -s /bin/bash apache -c "php -dmemory_limit=512M ./occ $params"
+    TERM=dumb runuser -s /bin/bash apache -c "php -d memory_limit=1024M -d apc.enable_cli=1 ./occ $params"
 }
 
 exitOnError () {
     echo $1
-    OCC "app:enable accessibility"
```

Comment:
maybe we could enable acessibility but lock nextcloud in maintenance

Reply:
ok, I will do it